EIPS/eip-modexp-limit.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: nnnn
+title: Set upper bounds for MODEXP
+description: Each input field is restricted to a maximum of 8192 bits
+author: Alex Beregszaszi (@axic), Radoslaw Zagorowicz (@rodiazet)
+discussions-to: 
```

Comment:
Will create this once an EIP number is assigned.

Reply:
Please use the template when creating a discussions topic in Eth Magicians: https://ethereum-magicians.org/c/eips/5

There isn't a need to wait for a number to be assigned.  I have been updating the titles on Eth Magicians with the number.

Reply:
I've seen the template, but also less work to create it after the number was assigned, as no editing needed 😀

Reply:
Added.